doc-content/enterprise-only/installation/nfs-server-configure_proc.adoc

```diff
@@ -0,0 +1,20 @@
+[id='nfs-server-configure_{context}']
+= Configuring an NFS server
+
+You must deploy and configure an NFS server to provide the file system necessary for a {CENTRAL} clustered environment.
+
+You must use NFS version 4.
+
+.Procedure
+
+. Configure a server to export NFS version 4 shares. For instructions about exporting NFS shares on Red Hat Enterprise Linux, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_file_systems/exporting-nfs-shares_managing-file-systems[Exporting NFS shares].
+
+. Create a `/data/shared` share with the options: `rw,sync,no_root_squash`. For example, you can use one of the following lines in the `/etc/exports` file:
++
+[subs="attributes,verbatim,macros"]
+----
+/data/shared *(rw,sync,no_root_squash)
```

Comment:
just to be consistent with directories I would suggest to change those to "/opt/kie/data". And also I would remember people that they can use any directory they want, they just need to remember to use them when mounting the shared FS and of course replacing it in `/etc/hosts`